app/javascript/gobierto_dashboards/webapp/containers/shared/Home.vue

```diff
@@ -21,25 +25,56 @@
 <script>
 import Nav from "./Nav.vue";
 import Aside from "./Aside.vue";
+import Summary from "./../summary/Summary.vue";
+import ContractsIndex from "./../contract/ContractsIndex.vue";
+import ContractsShow from "./../contract/ContractsShow.vue";
+import TendersIndex from "./../tender/TendersIndex.vue";
+import TendersShow from "./../tender/TendersShow.vue";
 
+import { EventBus } from "../../mixins/event_bus";
 import { store } from "../../mixins/store";
 
 export default {
   name: 'Home',
   components: {
     Aside,
-    Nav
+    Nav,
+    Summary,
+    ContractsIndex,
+    ContractsShow,
+    TendersIndex,
+    TendersShow
   },
   data() {
     return {
       activeTabIndex: store.state.currentTab || 0,
+      contractsData: this.$root.$data.contractsData,
     }
   },
+  computed: {
+    isSummary: function() { return this.$route.name === 'summary' },
+    isContractsIndex: function() { return this.$route.name === 'contracts_index' },
+    isContractsShow: function() { return this.$route.name === 'contracts_show' },
+    isTendersIndex: function() { return this.$route.name === 'tenders_index' },
+    isTendersShow: function() { return this.$route.name === 'tenders_show' },
```

Comment:
1. ES6
2. Are these values changing while the component is mounted? It looks like they don't

Reply:
1. I think you can't use arrow functions in computed properties, or do you mean something else?

2. I think they are changing. Home is mounted just once and every time the tab changes these properties change too.

Reply:
I'm just saying so, keep consistency in the code format

```suggestion
    isSummary() { return this.$route.name === 'summary' },
    ...etc
```

Reply:
Ahh. Understood, thanks!